sky/clouds/oci.py

```diff
@@ -281,40 +267,36 @@ def _make(instance_list):
             clouds='oci')
         if instance_list is None:
             return ([], fuzzy_candidate_list)
-        
+
         return (_make(instance_list), fuzzy_candidate_list)
-    
 
-    def check_credentials(self) -> Tuple[bool, Optional[str]]:
+    @classmethod
+    def check_credentials(cls) -> Tuple[bool, Optional[str]]:
         try:
-            user = oci_conf.identity_client.get_user(oci_conf.oci_config["user"]).data
-            logger.info("* check_credentials passed.")
+            user = oci_conf.identity_client.get_user(
+                oci_conf.oci_config['user']).data
+            logger.info('* check_credentials passed.')
```

Comment:
For the logging info here, since we will show the output in `sky check`, we may want to make sure the output looks concise and align with the other clouds. : )

Reply:
For log msg, i will review and format them later